bower.json

```diff
@@ -5,7 +5,7 @@
     "url": "https://github.com/emberjs/ember.js.git"
   },
   "dependencies": {
-    "jquery": ">=1.11.1 <3.0.0",
```

Comment:
I do not think we are using private API's after this change, perhaps this should just be `>= 1.11.1 < 4.0.0`?